follower/consensus_follower.go

```diff
@@ -45,7 +45,6 @@ func WithBootstrapDir(bootstrapDir string) Option {
 
 func getAccessNodeOptions(config *Config) []access.Option {
 	return []access.Option{
-		access.WithUpstreamAccessNodeID(config.upstreamAccessNodeID),
```

Comment:
Can you either update `ConsensusFollower` with the new arguments that are required instead, or create an issue for it? Thanks :)

Reply:
I created an issue https://github.com/onflow/flow-go/issues/1126 if you don't want to address this in the current PR.

Reply:
I actually went ahead and changed the `consensus_follower` to accept a addresses and keys instead a single id.